autopush/websocket.py

```diff
@@ -1090,7 +1094,8 @@ def ver_filter(update):
             size = len(msg.data) if msg.data else 0
             log.msg("Ack", router_key="webpush", channelID=chid,
                     message_id=version, message_type="direct",
-                    message_size=size, uaid_hash=self.ps.uaid_hash)
+                    message_size=size, uaid_hash=self.ps.uaid_hash,
+                    **{'user-agent': self.ps.user_agent})
```

Comment:
Ah, I thought you wanted this to match how we're logging in endpoint. There, it's going in as "user-agent" thanks to pretty much the same thing via client_info.

Perfectly happy to use user_agent if that's not a problem with the logging stuff.